src/components/avatar-volume-controls.js

```diff
@@ -80,6 +84,10 @@ AFRAME.registerComponent("avatar-volume-controls", {
     const step = -calcGainStepDown(gainMultiplier);
     gainMultiplier = THREE.MathUtils.clamp(gainMultiplier + step, 0, MAX_GAIN_MULTIPLIER);
     this.updateGainMultiplier(gainMultiplier, true);
+    // If the gainMultiplier is lowered to 0, updated muted status in local storage
```

Comment:
This is already calling `updateGainMultiplier` which already calls `updateLocalMuted` so I believe we can remove the `updateLocalMuted` call from here.